addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -367,18 +369,35 @@ def _l10n_es_tbai_get_invoice_values(self, invoice, cancel):
 
         return values
 
+    def _l10n_es_tbai_get_invoice_line_price_total(invoice_line):
```

Comment:
```suggestion
    def _l10n_es_tbai_get_invoice_line_price_total(self, invoice_line):
```
See ci/style as well.

Reply:
Sorry, it's done, all checks have passed, thank you !